lib/muser.h

```diff
@@ -413,16 +422,6 @@ lm_ctx_destroy(lm_ctx_t *lm_ctx);
 int
 lm_ctx_drive(lm_ctx_t *lm_ctx);
 
-/**
- * Creates and runs an lm_ctx.
- *
- * @dev_info: device information used to create the context
- *
- * @returns 0 on success, -1 on failure. Sets errno.
- */
-int
-lm_ctx_run(lm_dev_info_t *dev_info);
-
```

Comment:
thanks. do we need lm_get_region() still?

Reply:
I am not sure about this. Will keep it, added FIXME to investigate.